modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/parser/helper/DefinitionParserHelper.java

```diff
@@ -677,7 +677,7 @@ private static OptionHolder constructOptionProcessor(StreamDefinition streamDefi
         Map<String, String> options = new HashMap<String, String>();
         Map<String, String> dynamicOptions = new HashMap<String, String>();
         for (Element element : annotation.getElements()) {
-            if (Pattern.matches("\\{\\{.*?\\}\\}", element.getValue())) {
+            if (Pattern.matches("(.*?)\\{\\{.*?}}(.*?)", element.getValue())) {
```

Comment:
Hi Minudika,

\\}\\} has to be there else it will not work on Android! 

Please fix that ASAP.